tools/db_crashtest.py

```diff
@@ -594,7 +574,30 @@ def finalize_and_sanitize(src_params):
     if dest_params.get("create_timestamped_snapshot_one_in", 0) > 0:
         dest_params["txn_write_policy"] = 0
         dest_params["unordered_write"] = 0
-
+    # WAL data can be lost when `manual_wal_flush_one_in > 0` and `FlushWAL()`` is not
+    # explictly called by users of RocksDB (in our case, db stress).
+    # Therefore the recovery from such potential WAL data loss is a prefix recovery
+    # that requires `sync_fault_injection=1` to verify.
```

Comment:
As mentioned above, while technically this is true today, the condition for enabling tracing could be expanded to make `sync_fault_injection=1` not necessary here.

Reply:
Will fix.

Reply:
Fixed